collector/src/main/java/org/apache/hertzbeat/collector/collect/jmx/JmxCollectImpl.java

```diff
@@ -73,6 +74,7 @@ public class JmxCollectImpl extends AbstractCollect {
 
     public JmxCollectImpl() {
         connectionCommonCache = new ConnectionCommonCache<>();
+        Thread.currentThread().setContextClassLoader(new ClassLoaderConfig(ClassLoader.getSystemClassLoader()));
     }
```

Comment:
Hi, this seem can not work due the constructor new JmxCollectImpl() is call by the thread-A and  JmxCollectImpl is singleton, but the `JmxCollectImpl.collect` is run by another thread-B in running-thread-pool.  `Thread.currentThread().setContextClassLoader` only can take effect on the current thread.